src/App.tsx

```diff
@@ -1,24 +1,215 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { FormEvent, useEffect, useState } from 'react';
+import { Todo } from './Components/Todo';
+import { Todo as TodoType } from './types/Todo';
+import {
+  getTodos,
+  addTodo,
+  removeTodo,
+  updateTodo,
+} from './api/todos';
+import { Notification } from './Components/Notification';
+import { Filter } from './Components/Filter';
 
-const USER_ID = 0;
+const USER_ID = 10539;
```

Comment:
USER_ID should not be hardcoded as a constant. Consider using a more dynamic way to get the user ID or a more meaningful constant name.